openstack/octavia/templates/octavia-api-ingress.yaml

```diff
@@ -11,10 +11,11 @@ metadata:
     system: openstack
     type: api
     component: octavia
-{{- with .Values.ingress.annotations }}
   annotations:
-{{ toYaml . | indent 4 }}
-{{- end }}
+    # linkerd annotations
+    {{- include "utils.linkerd.ingress_annotation" . | indent 4 }}
+    # other annotations
+    {{- .Values.ingress.annotations | toYaml | nindent 4 }}
```

Comment:
With this change, you cannot unset `.Values.ingress.annotations` anymore. This is a change to the previous behavior. Could be fine, because we have defaults for that. The chart renders unless linkerd is also enabled afaics.